docs/website/docs/walkthroughs/create-a-pipeline.md

```diff
@@ -1,31 +1,46 @@
 ---
 title: Create a pipeline
 description: How to create a pipeline
-keywords: [how to, create a pipeline]
+keywords: [how to, create a pipeline, rest client]
 ---
 
 # Create a pipeline
 
-Follow the steps below to create a [pipeline](../general-usage/glossary.md#pipeline) from the
-WeatherAPI.com API to DuckDB from scratch. The same steps can be repeated for any source and
-destination of your choice—use `dlt init <source> <destination>` and then build the pipeline for
-that API instead.
+This guide walks you through creating a pipeline that uses our [REST API Client](../general-usage/http/rest-client)
+to connect to [DuckDB](../dlt-ecosystem/destinations/duckdb).
+Although this example uses DuckDB, you can adapt the steps to any source and destination by
```

Comment:
Let's link to from "source" and "destination" to respective sections in the integrations:
https://dlthub.com/docs/dlt-ecosystem/verified-sources/
https://dlthub.com/docs/dlt-ecosystem/destinations/

Reply:
Done.